content/account/dashboard/index.textile

```diff
@@ -0,0 +1,116 @@
+---
+title: Overview
+meta_description: " Manage and monitor your applications on the Ably platform using the Ably dashboard. Create new apps, view existing ones, and configure settings from your browser."
+meta_keywords: “Ably dashboard, manage applications, monitor apps, app creation, application settings, Ably platform”
+---
+
+Begin by signing in to Ably through your browser. Once you're signed in, you have access to the Ably "dashboard":https://ably.com/accounts, where you can "create news app":#create or "view your existing apps.":#view
+
+h2(#create). Create a new app
+
+The image below displays the Ably dashboard when no apps have been created yet. To get started, you can easily *create a new app*:
+
+<a href="@content/screenshots/dash/create-app.png" target="_blank">
+  <img src="@content/screenshots/dash/create-app.png" style="width: 100%" alt="Create apps in the Ably dashboard">
+</a>
+
+h2(#view). View your apps
+
+If you have created apps already, your dashboard will look similar to the one shown in the following image. Each app shown is a link to its specific dashboard allowing you to view and configure the settings, as well as view usage and statistics:
+
+<a href="@content/screenshots/dash/your-apps.png" target="_blank">
+  <img src="@content/screenshots/dash/your-apps.png" style="width: 100%" alt="Your apps in the Ably dashboard">
+</a>
+
+h2. Dashboard overview
+
+The following tabs in your app dashboard enable you to monitor and configure your application:
+
+<a href="@content/screenshots/dash/dash-tabs.png" target="_blank">
+  <img src="@content/screenshots/dash/dash-tabs.png" style="width: 100%" alt="Your app dashboard tabs">
+</a>
+
+h2. App statistics
```

Comment:
Should these be H3 after the overview?

I also think header names should match the tab names.